src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -112,8 +110,17 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             int retry_num = 0;
             while(true)
             {
+                ChromeDriverService? driverService = null;
                 try
                 {
+                    var driverLogPath = Path.Combine(_arguments.OutputDirectory, $"chromedriver-{retry_num}.log");
+                    File.Delete(driverLogPath);
```

Comment:
does this need to be here now that the logs are numbered?

Reply:
Just useful when doing multiple runs locally, so that you are not really looking at a stale file. For example, if `chromedriver` cannot be found then the older log file won't get overwritten. Though maybe, we should delete `chromedriver-*.log` before we start, so be completely clear of stale log files?